src/java/picard/util/ScatterIntervalsByNs.java

```diff
@@ -33,12 +33,24 @@
  */
 
 @CommandLineProgramProperties(
-        usage = "Writes an interval list based on splitting the reference by Ns.",
-        usageShort = "Writes an interval list based on splitting the reference by Ns",
+        usage = ScatterIntervalsByNs.USAGE_SUMMARY + ScatterIntervalsByNs.USAGE_DETAILS,
+        usageShort = ScatterIntervalsByNs.USAGE_SUMMARY,
         programGroup = Intervals.class
 )
 public class ScatterIntervalsByNs extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "A CLP for breaking up a reference into intervals of Ns and ACGTs bases.  ";
```

Comment:
"A CLP for breaking up" -> "Breaks up"

CLP just stands for command line program, but it is confusing for users.